api/pkg/apps/webapp/templates/individual.gohtml

```diff
@@ -349,23 +349,27 @@
                     let partySelectorList = document.getElementById("partySelectorList")
                     partySelectorList.innerHTML = ''
                     if (res.response.items.length > 0) {
+                        let n = 0
                         res.response.items.forEach(party => {
+                          if (n < 5) {
```

Comment:
I think we should avoid using "magic numbers" as much as possible, it makes the code harder to maintain.
What about creating a `Constants` struct in the webapp and passing it to the renderer? Then having something like `if (n < {{.Constants.NumPartyDropdownItems}})`

Reply:
That makes sense, will make the change and re-request review!